app/models/validations/date_validations.rb

```diff
@@ -57,6 +63,14 @@ def first_collection_start_date
     @first_collection_start_date ||= FormHandler.instance.forms.map { |form| form.second.start_date }.compact.min
```

Comment:
Do you need this line now you've added the memoization method below?

Reply:
I'm pretty sure we still need this one, otherwise after we remove the 2021 form someone could enter any start date in the past and it would pass validation.